rsocket/RSocketServer.cpp

```diff
@@ -135,7 +135,6 @@ void RSocketServer::onRSocketSetup(
   auto responder = std::make_shared<ScheduledRSocketResponder>(
       std::move(connectionParams.responder), *eventBase);
   auto rs = std::make_shared<RSocketStateMachine>(
-      *eventBase,
```

Comment:
nit: SetupResumeAcceptor holds an instance of `eventBase`. 
Instead of getting the `eventbase` using the `getExistingEventBase` function, we might make `SetupResumeAcceptor` to call `onSetup` function with the `eventBase` that it holds. 
It might remove one more ambiguity of which event base do we get from the EventBaseManager, which is clear that it is the event base of the io thread, but..